src/components/prescriptions/PrescriptionButtonComponent.vue

```diff
@@ -0,0 +1,284 @@
+                    >
+                    <input
+                        id="addManually"
+                        v-model="researchDate"
+                        type="date" />
+                </div>
+            </div>
+
+            <div v-show="importFlag">
+                <div class="inputBox">
+                    <label for="uploadTest">Dodaj wyniki:</label>
+                    <div
+                        class="icon"
+                        tabindex="0">
+                        <font-awesome-icon icon="fa-solid fa-file-import" />
+                    </div>
+                    <input
+                        id="uploadTest"
+                        class=""
+                        type="file"
+                        @change="handleChange" />
+                    <error-info :message="testFileError" />
+                </div>
+            </div>
+            <div class="actionsAndInfo">
+                <button-component
+                    :disabled="disableAddTest"
+                    wide
+                    >Zapisz receptę</button-component
+                >
+                <bouncing-balls-component :visible="isLoading" />
+                <error-info :message="error" />
+                <span
+                    v-show="successFlag"
+                    aria-label="passwords match"
+                    class="dataSend"
+                    role="presentation"
+                    >Recepta została przesłana pomyślnie</span
+                >
+            </div>
+        </form>
+    </div>
+</template>
+
+<script setup>
+    import ButtonComponent from '../ButtonComponent.vue';
+    import BouncingBallsComponent from '../BouncingBallsComponent.vue';
+    import ErrorInfo from '../ErrorInfo.vue';
+    import getDoctorsArray from '../../composables/getDoctor.js';
+    import getMedicineArray from '../../composables/getMedicine.js';
+
+    import { computed, ref } from 'vue';
+    import ALLOWED_TYPES from '../../utils/allowedTypes.js';
+    import useStorage from '../../composables/getImage.js';
+    import useUserStore from '../../stores/userStore.js';
+    import { timestamp } from '../../firebase/config.js';
+    import useCollection from '../../composables/useCollections.js';
+    const { isLoading, addDoc, error } = useCollection('listOfRecipies');
+    const { url, uploadImage } = useStorage();
+
+    const addManually = ref(false);
+    const importFlag = ref(null);
+    const nameAndSurnameDoctor = ref(null);
+    const medicine = ref(null);
+    const frequency = ref(null);
+    const researchDate = ref(null);
+    const successFlag = ref(false);
+    const testFile = ref(null);
+    const testFileError = ref(null);
+
+    const userStore = useUserStore();
+
+    const disableAddTest = computed(
+        () => !(dateFormat.value || testFile.value)
+    );
+
+    const doctorNameAndSurname = computed(() => nameAndSurnameDoctor.value);
+
+    const medicineNames = computed(() => medicine.value);
```

Comment:
this is unecessary cause you do not make any computing in computed function. This statement now is something like that: `let a = 1; b = 1 `